Yosemite/Yosemite/Stores/Order/OrdersUpsertUseCase.swift

```diff
@@ -72,10 +72,6 @@ struct OrdersUpsertUseCase {
         let siteID = readOnlyOrder.siteID
         let orderID = readOnlyOrder.orderID
 
-        guard readOnlyOrder.items.count > 0 else {
```

Comment:
I'm curious, are we sure that removing this code won't add any regression? I see the commit message from @pmusolino when added with the message 

> fix: handle order items in OrdersUpsertUseCase

Reply:
I was looking at the code below that has the comment
`// Now, remove any objects that exist in storageOrder.items but not in readOnlyOrder.items`
And if we do not remove this `guard` then the removing of the objects in storage might not happen?